config/master/aws-k8s-cni-configmap.yaml

```diff
@@ -0,0 +1,27 @@
+---
+"apiVersion": "v1"
+"data":
```

Comment:
Initially, do we want to keep this ConfigMap in the same file as the daemonset config, or do we want to keep them in separate files? There are upsides and downsides with both I guess.

This will also be related to how we handle it in eks-charts.

Reply:
@mogren I like to have configmap as separate file. By having it as a separate file, we will not undo all the changes made by the customer in their previous deployments if we have it in a single file.